tags/images-slider.tag.html

```diff
@@ -169,6 +169,22 @@
         function loadIntoWorkArea(e){
             imgSelected = tag.thumbnails[e.item.index];
             riot.mount("workarea",{ img : imgSelected });
+            readyToOpacity()
```

Comment:
This file is not the correct place. You should do the change in `workarea.tag.html`. In addition, a component will be loaded only page is loaded so you need not to check `document.readyState`